stripe_terminal/lib/src/models/tap_to_pay_ux_configuration.dart

```diff
@@ -0,0 +1,60 @@
+/// The `TapToPayUxConfiguration` class is used to configure the UX of the Tap to Pay flow.
+///
+/// Do not create new instances of this class. Instead, set the properties via [Terminal.setTapToPayUxConfiguration].
+class TapToPayUXConfiguration {
+  TapToPayUxConfigurationTapZone? tapZone;
+  TapToPayUxConfigurationColors? colors;
+  TapToPayUxConfigurationTheme? theme;
+
+  TapToPayUXConfiguration({
+    this.tapZone,
+    this.colors,
+    this.theme,
+  });
+}
+
+class TapToPayUxConfigurationTapZone {
+  TapToPayUxConfigurationTapZoneIndicator? indicator;
+  TapToPayUxConfigurationTapZonePosition? position;
+
+  TapToPayUxConfigurationTapZone({
```

Comment:
```suggestion
class TapToPayUxConfigurationTapZone {
  final TapToPayUxConfigurationTapZoneIndicator? indicator;
  final TapToPayUxConfigurationTapZonePosition? position;

  const TapToPayUxConfigurationTapZone({
```
Missing final and const